Add middle eastern chili recipe (#24)
Shrinks99/recipes: content/soups and stews/Middle-Eastern Chili.md

---
title: Middle-Eastern Chili
category: soups and stews
date: 2025-08-21
recipeYear: "2025"
recipeDecade: 2020s
author: Liza Desyatkova
prepTime: 15m
cookTime: 30m
vegetarian: true
---

### Ingredients

> #### Part 1

Reviewer: Part headings are not required if the instructions are not split into multiple parts

- 1 16 ounce can chickpeas, rinsed and drained
- 1 16 ounce can fava beans beans, rinsed and drained
- 1 28 ounce can crushed tomatoes
- 1 12 ounce can of corn
- 2 1/2 cups soup broth of choice
- 1 small red bell pepper diced
- 1 medium yellow onion diced
- 1 small lemon, juiced
- 4 cloves minced garlic
- 2 tablespoons of olive oil
- 2 tablespoons chili powder
- 1 tablespoon curry powder 
- 1 tablespoon paprika
- 3 teaspoon dried cumin
- 2 teaspoons ground coriander
- 2 teaspoons of ground masala
- 1 teaspoon ground cardamom
- 1 teaspoon ground cloves
- 1 teaspoon ground allspice
- 2 bay leafs
- Bread of choice
- **Optional:** BBQ or Dijon sauce for extra flavour

---

### Instructions

> 1. Prepare large pot with olive oil on medium heat 

Reviewer: Please remove trailing spaces at the end of lists.  Instructional lists should also have periods at the end.

```suggestion
1. Prepare a large pot with olive oil over medium heat.
```

2. Saute the diced onion, diced bell pepper, and minced garlic for 5 minutes
3. Add all the spices, saute for another 3 minutes
4. Add crushed tomatoes, cook for 5 minutes
5. Add chickpeas, favabeans, corn, and soup broth
6. Optional: add BBQ sauce, Dijon mustard, or your favourite sauce
7. Cook until chili boils down to desired viscosity
8. Add lemon juice
9. Serve with warm bread or naan

> ### Comments

Reviewer: Please add a newline before headers

> You can skip buying the spices you don't have, it will be delicious either way. 

Reviewer: ```suggestion
You can skip buying the spices you don't have any, it will be delicious either way. 
```

Bonus trick: if you have whole cardamom, cloves, or cumin seeds, turn them into powder using a seed grinder, or mortar & pestle. This makes a huge difference in dish fragrance. 
### Historical Record

> This recipe was adapted for Liza's hardcore camping trip. She referenced this [recipe](https://www.fooddolls.com/middle-eastern-chili-recipe/) with some alterations of removing the meat, and adding more of her favourite spices. 

Reviewer: Generally linking to external resources alone as a form of attribution is sorta half discouraged.  This repo is intended for long-term preservation, wherever possible include more attribution info in-text!  Websites go away! 🥲 

```suggestion
This recipe was adapted for Liza's hardcore camping trip. She referenced [Food Dolls' _Middle Eastern Chili Recipe_](https://www.fooddolls.com/middle-eastern-chili-recipe/) with some alterations of removing the meat, and adding more of her favourite spices. 
```
